### PR TITLE
Add support for telugu language

### PR DESCRIPTION
**Pull request by kiranchavala**
This PR adds support for Telugu language in the Cloudstack Ui 

Screenshot 2024-11-20 at 4 55 17 PM

### CODE REVIEW

File: ui/package-lock.json
Comment (DaanHoogland) [inline ui/package-lock.json:1]: I think you need to remove this change

**Review by sureshanaparti — APPROVED** [review]
clgtm

**Comment by sureshanaparti** [comment]
@blueorangutan package

File: ui/package-lock.json
Comment (kiranchavala) [reply ui/package-lock.json:1]: removed the change

**Comment by kiranchavala** [comment]
@blueorangutan ui

**Review by DaanHoogland — APPROVED** [review]
lgtm (no native telugu speaker!)

File: ui/public/locales/te.json [hunk 1]
```diff
@@ -0,0 +1,3737 @@
...
+  "VM_SNAPSHOT": "VM స్నాప్‌షాట్",
+  "VM_SNAPSHOT_ON_PRIMARY": "ప్రాథమికంపై VM స్నాప్‌షాట్",
+  "VOLUME": "వాల్యూమ్",
+  "VOLUME_SECONDARY": "సెకండరీలో వాల్యూమ్",
+  "VPN_USERS": "VPN వినియోగదారులు",
+  "Compute*Month": "గణన * నెల",
+  "GB*Month": "GB * నెల",
+  "IP*Month": "IP * నెల",
+  "Policy*Month": "విధానం * నెల"
+}
```
Comment (DaanHoogland) [inline ui/public/locales/te.json:3737]:
```suggestion
}

```

**Review by rohityadavcloud — APPROVED** [review]

**Comment by DaanHoogland** [comment]
@kiranchavala , please resolve the lint issue.

**Comment by shwstppr** [comment]
@blueorangutan ui

**Comment by rohityadavcloud** [comment]
@JoaoJandre @DaanHoogland some PRs targeting 4.20.1 are merged on main - should they go to 4.20 branch?

**Comment by JoaoJandre** [comment]
@rohityadavcloud Considering that they were not targeting 4.19 (a maintenance branch), I assumed the authors wanted their changes to be made on the main branch directly. I'll fix the milestone. If the authors want their changes to be on the 4.20 branch, I volunteer to backport the PRs (https://github.com/apache/cloudstack/pull/9930, https://github.com/apache/cloudstack/pull/9708, https://github.com/apache/cloudstack/pull/9956).

**Pull request merged**